### DISCUSSION TITLE
Monthly depreciation 0 bug or not?

### DISCUSSION THREAD

**Discussion by jonas-noa** [snipe/snipe-it#10156] [Q&A]
Hi, 
I'm currently testing out Snipe-it. It is truely an amazing platform. I was testing the feature of **depreciation** because it was a very important element for the accountingteam. I noticed when I setup a depreciation (linear) over 60 months that the report shows the correct Diff value but the **"Monthly Depreciation" is always 0**. Am I missing something. I tried reproducing this in the online demo and noticed that the monthly depreciation is getting filled in. Is this a bug or did I forget to configure something?

Thank you for the feedback.

**Comment by snipe** [comment]
Thanks for the kind words - can you show me a screenshot of data where you're seeing this?

**Reply by jonas-noa** [reply]
Thank you for your fast reply. Sorry I forgot to upload. This is a snippet of the list showing under "Depreciation report"
![image](https://user-images.githubusercontent.com/67271006/135696971-0e0437f1-935d-4d93-afad-bb7041cda30e.png)

**Comment by snipe** [comment]
Do those assets have a purchase date?

**Comment by jonas-noa** [comment]
Yes. 
![image](https://user-images.githubusercontent.com/67271006/135697904-2e1d5a9a-c7f6-4d2a-9f0c-fb3f914c455a.png)

**Comment by jonas-noa** [comment]
I found the solution. It is not a bug I guess? I was reading the code "DepreciationReportTransformer.php" and at line 66 you have this:             `$monthly_depreciation = $purchase_cost_currency . ' ' .\App\Helpers\Helper::formatCurrencyOutput(($asset->model->eol > 0 ? ($asset->purchase_cost / $asset->model->eol) : 0));`

So the monthly depreciation depends on the EOL field of an assetmodel. I filled 0 in EOL of the assetmodel not knowing what the real meaning was of the EOL field. I thought this was the date the vendor doesn't support the asset anymore (like vendors label products EOL when the don't produce or update a certain product anymore). I didn't knew it was relevant for the monthly depreaciation. What is the recommended value to fill in? The duration of the warranty? Because (in Belgium) the assets like computers the warranty < then the duration of the depreciationperiod. 

Thank you (sorry for wasting your time)
